WcaOnRails/spec/models/registration_spec.rb

```diff
@@ -54,13 +54,13 @@
   end
 
   it "requires at least one event" do
-    registration.eventIds = ""
+    registration.registration_events.destroy_all
```

Comment:
I'm curious about this. `destory_all` means we've written to the database by the time we run our validations on the next line. But what's actually going to happen is more like this, right?

```ruby
registration.registration_events_attributes = []
expect(registration).to be_invalid
```

Fingers crossed that our validations can handle this....